<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>544</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>593</width>
    <height>544</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pics.qrc">
    <normaloff>:/pics/Logo_GNAL.png</normaloff>:/pics/Logo_GNAL.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>3.000000000000000</double>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>461</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>Historial</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>241</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Recibidos</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="recibido_lista">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="serie_radio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>true</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Enviados</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="dato_lista">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>81</width>
      <height>184</height>
     </rect>
    </property>
    <property name="title">
     <string>Datos</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLineEdit" name="dato_edit_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="maxLength">
          <number>2</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dato_edit_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="maxLength">
          <number>2</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dato_edit_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="maxLength">
          <number>2</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dato_edit_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="maxLength">
          <number>2</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dato_edit_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="maxLength">
          <number>2</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Registro</string>
    </property>
    <widget class="QLineEdit" name="dato_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::VisualMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="enviar_boton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>59</width>
      <height>58</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="pics.qrc">
      <normaloff>:/pics/icon-venue-access.png</normaloff>:/pics/icon-venue-access.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Conexión</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>132</width>
       <height>90</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="serie_combo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="serie_actualizar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="pics.qrc">
            <normaloff>:/pics/Custom-Icon-Design-Mono-General-4-Refresh.ico</normaloff>:/pics/Custom-Icon-Design-Mono-General-4-Refresh.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="conectar_boton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pics.qrc">
          <normaloff>:/pics/red-usb-disconnected-256.png</normaloff>:/pics/red-usb-disconnected-256.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="conf_boton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pics.qrc">
          <normaloff>:/pics/256-256-5fbc60a4335d01cd9c35dcf8fae02410.png</normaloff>:/pics/256-256-5fbc60a4335d01cd9c35dcf8fae02410.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="i2c_enviar">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ENVIAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="i2c_registro">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="i2c_dato">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="i2c_check">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>140</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>i2c</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>conf_boton</tabstop>
  <tabstop>dato_edit</tabstop>
  <tabstop>dato_edit_2</tabstop>
  <tabstop>dato_edit_3</tabstop>
  <tabstop>dato_edit_4</tabstop>
  <tabstop>dato_edit_5</tabstop>
  <tabstop>dato_edit_6</tabstop>
  <tabstop>enviar_boton</tabstop>
  <tabstop>recibido_lista</tabstop>
  <tabstop>dato_lista</tabstop>
 </tabstops>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dato_edit_6</sender>
   <signal>returnPressed()</signal>
   <receiver>enviar_boton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dato_edit_2</sender>
   <signal>returnPressed()</signal>
   <receiver>enviar_boton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dato_edit_3</sender>
   <signal>returnPressed()</signal>
   <receiver>enviar_boton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dato_edit_5</sender>
   <signal>returnPressed()</signal>
   <receiver>enviar_boton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dato_edit_4</sender>
   <signal>returnPressed()</signal>
   <receiver>enviar_boton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enviar_boton</sender>
   <signal>clicked()</signal>
   <receiver>dato_edit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enviar_boton</sender>
   <signal>clicked()</signal>
   <receiver>dato_edit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dato_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>enviar_boton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>dato_edit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>conf_boton</sender>
   <signal>clicked()</signal>
   <receiver>dato_edit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>conf_boton</sender>
   <signal>clicked()</signal>
   <receiver>dato_edit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
